shared/teams/team/container.js

```diff
@@ -179,18 +182,19 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
 
   const you = stateProps.you
   let youExplicitAdmin = false
-  let youImplicitAdmin = false
+  let youImplicitAdmin = stateProps.yourOperations.manageMembers
   let youAreMember = false
   if (you) {
-    youExplicitAdmin = Constants.isOwner(stateProps.yourRole) || Constants.isAdmin(stateProps.yourRole)
-    youImplicitAdmin = stateProps._implicitAdminUsernames.has(you)
```

Comment:
don't you still need this line? Is `manageMembers` false if you're an explicit admin of the team?

Reply:
good catch, thanks. Will we need a way to distinguish just from the operations flags?

Reply:
talked over out loud: adding `shouldShowAddYourselfBanner` (or similar) to `canUserPerform` for this